packages/access-client/src/types.ts

```diff
@@ -103,7 +107,7 @@ export interface Service {
     info: ServiceMethod<SpaceInfo, SpaceRecord, Failure | SpaceUnknown>
     'recover-validation': ServiceMethod<
       SpaceRecoverValidation,
-      EncodedDelegation<[SpaceRecover]> | undefined,
+      ShouldShowValidationNonce | EncodedDelegation<[SpaceRecover]> | undefined,
```

Comment:
```suggestion
      ShouldShowValidationNonce | undefined,
```

Reply:
hm is this suggestion right or did you mean to suggest removing `ShouldShowValidationNonce` ?